soc/arm/renesas_smartbond/da1469x/soc.c

```diff
@@ -31,4 +49,75 @@ static int renesas_da14699_init(void)
 	return 0;
 }
 
+#if defined(CONFIG_HAS_FLASH_LOAD_OFFSET)
+static void z_renesas_configure_cache(void)
```

Comment:
@carlocaione

Reply:
What cache is this and what this code is doing? Asking to check whether this can be hooked up to the cache API instead of having soc-specific code.

Reply:
Hi @carlocaione,

The DA1469x has execute in place from external flash, this function configures the cache controller for the XIP mechanism. It also copies the IVT to RAM. Configuring the cache needs a software reset too. I don't think the cache driver is a good fit/ or that other soc's will use this specific mechanism.

Reply:
yeah, I agree, this is fine then.